src/ConnectedMode/Suppressions/SuppressedIssueMatcher.cs

```diff
@@ -75,7 +74,7 @@ private static bool IsMatch(IFilterableIssue issue, SonarQubeIssue serverIssue)
                 return false;
             }
 
-            if (!IsFileMatch(issue, serverIssue))
+            if (!PathHelper.IsServerFileMatch(issue.FilePath, serverIssue.FilePath))
```

Comment:
We need the same file-comparison logic when checking Roslyn suppressions, so I've moved the method to `Core.PathHelper`.